check.sh

```diff
@@ -23,6 +23,12 @@ flake8 trio/ \
     --ignore=D,E,W,F401,F403,F405,F821,F822\
     || EXIT_STATUS=$?
 
+# Run mypy
+# We specify Linux so that we get the same results on all platforms. Without
```

Comment:
Can you edit this comment to mention your future plans listed below? (run mypy on all platforms, use assert guards to avoid typechecking the wrong IOManagers)

Reply:
Thanks, done